modules/claims_api/app/sidekiq/claims_api/flash_updater.rb

```diff
@@ -2,12 +2,22 @@
 
 require 'sidekiq'
 require 'sentry_logging'
+require 'claims_api/claim_logger'
 
 module ClaimsApi
   class FlashUpdater < UpdaterService
     include Sidekiq::Job
     include SentryLogging
 
+    sidekiq_retries_exhausted do |message|
+      ClaimsApi::Logger.log(
+        'claims_api_retries_exhausted',
+        claim_id: message['args'].last,
```

Comment:
is this consistent? and what's a risk here for future state for this to _not_ work? I think we only pass GUID's here, but can we protect against this in any way or is this something we'll just need to be aware of? @rockwellwindsor-va

Reply:
@FonzMP  each sidekiq job typically gets the ID as it's first value however with this job there are three params passed in and the ID is the third value (last).  Ideally going forward we would make sure that the ID is the first value, which is hopefully what testing for that feature would highlight when building anything new.  This job also has its own parent as I am sure you noticed when looking at the file, this and special issues have their own right block now because refactoring that I felt was probably too far out of scope for this, if we wanted to include it in the future we would need to either change the order the jobs receives its params in or check for that n the base service (probably based on the class value in the message)

Reply:
I didn't realize this was in a specific job. oversight there. I misread this as the standard for all jobs.